### ISSUE TITLE
Missing error on `.not()` notation

### ISSUE THREAD

**Issue by Volker-E** [stylelint/stylelint#4484]
When applying a mis-typed pseudo-class selector like `.not(.class)` instead of valid `:not(.class)` no error is thrown.

```css
a.not(.image) {
  border-bottom: 1px solid #aaa;
}
```
Origin: https://gerrit.wikimedia.org/r/#/c/mediawiki/skins/Vector/+/557153/1/print.less

> Which version of stylelint are you using?

`11.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

Via grunt-stylelint

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

It is reproducible in both, CSS and LESS.

> What did you expect to happen?

Errors to be flagged.

**Comment by evilebottnawi** [comment]
We do not check syntax of CSS, you can create plugin for it

**Comment by hudochenkov** [comment]
We try not to validate CSS in core rules. As @evilebottnawi said, it's better to be done in a plugin.

**Comment by jeddy3** [comment]
@Volker-E You can use [`csstree-validator`](https://github.com/csstree/validator) alongside stylelint to validate syntax.
